nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -307,7 +307,6 @@ let
         server {
           ${concatMapStringsSep "\n" listenString hostListen}
           server_name ${vhost.serverName} ${concatStringsSep " " vhost.serverAliases};
-          ${acmeLocation}
           ${optionalString (vhost.root != null) "root ${vhost.root};"}
```

Comment:
```suggestion
          ${optionalString (!vhost.forceSSL) acmeLocation}
          ${optionalString (vhost.root != null) "root ${vhost.root};"}
```

Reply:
Lets not even complicate it further just to support this case. The PR should probably be dropped as a whole.

Reply:
Thanks, fixed.